client/src/main/java/se/fortnox/reactivewizard/client/Response.java

```diff
@@ -37,7 +37,23 @@ public HttpResponseStatus getStatus() {
      * @return the value of the header or null if the header is missing
      */
     public String getHeader(String header) {
-        return headers.get(header);
+        return httpClientResponse
+            .responseHeaders()
+            .get(header);
+    }
+
+    /**
+     * Fetching a cookie value
+     * @param cookieName the cookie name
+     * @return the value(s) of the cookie
+     */
+    public List<String> getCookie(String cookieName) {
+        return httpClientResponse
+            .cookies()
+            .get(cookieName)
```

Comment:
Will get return null or an empty set if there is no cookie with that name? A test for this would be good.